LICENSE.md

```diff
@@ -73,19 +79,19 @@ The User may examine the Mod's source code.
 The Author reserves the right to distribute the Mod.  
 The Author may designate a person or organization as a Distributor.  
 A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
+The Author may require that a Distributor provides a direct link to the Mod.  
```

Comment:
I'm interested in your thought process relating to this change. Please explain what the difference between "codebase" and "mod" does, and why it's a good thing.